libcodechecker/server/api/report_server.py

```diff
@@ -53,6 +53,52 @@ class CountFilter:
     RUN_HISTORY_TAG = 6
 
 
+class DBSession():
+
+    def __init__(self, session_maker):
+        self.__session = None
+        self.__session_maker = session_maker
+
+    def __enter__(self):
+        # create new session
+        self.__session = self.__session_maker()
+        return self.__session
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        if self.__session:
+            self.__session.commit()
```

Comment:
I think it is not a good approach to commit automatically. For example when I query some data from the database, I do not want to commit anything.
Better solution can be to create a separate function for it in the  `DBSession` context manager class.